kotlin/rules.bzl

```diff
@@ -66,7 +66,7 @@ def _kotlin_compile_impl(ctx):
 
     if jars:
         # De-duplicate
-        jarsetlist = list(set(jars))
+        jarsetlist = list(depset(jars))
```

Comment:
The newer paradigm for making a list from a depset is `depset(jars).to_list()`.  Can we use that instead?

Reply:
sorry. I will fix it